zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -24,6 +24,28 @@ function HeliumCtrl($scope, $rootScope, $sce, baseUrlSrv, ngToast, heliumService
   $scope.showVersions = {};
   $scope.bundleOrder = [];
   $scope.bundleOrderChanged = false;
+  $scope.npmRegistry = 'http://registry.npmjs.org/'
+  $scope.mvnCentralRepository = 'http://repo1.maven.org/maven2/'
```

Comment:
@felixcheung Thanks for taking a look. I removed those links since they're not used anymore :)